Alex_RTOS_Sailboat/RTOS_framework_v1/src/sail_gps.c

```diff
@@ -0,0 +1,344 @@
+/*
+ * sail_gps.c
+ *
+ * Created: 2022-06-02 8:04 PM
+ *  Author: secul
```

Comment:
Add your name :)

Reply:
Added my name. That's my bad, secul is my admin username. I will watch out for this in subsequent file creations.